app/base/views.js

```diff
@@ -15,7 +15,7 @@ exports.home = function (req, res) {
       .catch(function (error) { render({'error': error}); });
 
   } else {
-    render({'user': {}});
+    res.redirect('/login');
```

Comment:
This could be done by adding `ensureLoggedIn('/login')` to the route, eg:
```javascript
var ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn;

exports.home = [
  ensureLoggedIn('/login'),
  function (req, res) {

    function render(context) { ... }

    api.authenticate(...);
    api.users.get(req.user.sub)
      .then(...)
      ...
  }
];